lib/src/di/injector.dart

```diff
@@ -53,20 +55,25 @@ void injectDependencies(Environment flavor) {
   _locator.registerSingleton<AppConfig>(AppConfig());
   _locator.registerSingleton<AppInfoRepository>(AppInfoLocalRepository());
   _locator.registerSingleton<PersistenceRepository>(SharedPreferencesPersistenceRepository());
-  _locator.registerSingleton<SettingsRepository>(SettingsLocalRepository(_locator<PersistenceRepository>()));
-  _locator.registerSingleton<PresentationShowingRepository>(PresentationShowingLocalRepository(_locator<PersistenceRepository>()));
+  _locator.registerSingleton<SettingsRepository>(
```

Comment:
większość zmian w tym pliku wynika z ctrl + alt + L, niestety z racji, że w robo mamy inny threshhold zawijania linii niż tu to ciężko mi to uspójnić 😞

Reply:
nie boli :) możesz powiedzieć co tam masz poustawiane to ustawię tak samo, żeby z czasem każdy pull request nie zawierał więcej zmian formatowania niż tych tych faktycznych :D

Reply:
w takim razie możesz ustawić tak 
![image](https://user-images.githubusercontent.com/24854180/99908963-f2593e80-2ce5-11eb-863c-cbb6002df64f.png)